BuildEFTools.cmd

```diff
@@ -15,4 +15,4 @@ ECHO "Warning: Cannot find installed VS path."
 
 :BuildStep
 
-msbuild "%~dp0\EFTools.msbuild" /p:RunCodeAnalysisForEFTools=true /v:minimal /p:Configuration=Release /t:EnableSkipStrongNames;Clean;BuildDesigner /maxcpucount /nodeReuse:false %*
```

Comment:
this I think addresses my previous question about `SkipStrongNames.targets` import that was removed. Seems like this new target is replacing what we had earlier.